### DISCUSSION TITLE
In the mobile app, the user should not have to provide server paths for login

### DISCUSSION THREAD

**Discussion by juxeii** [immich-app/immich#5396] [General]
Small improvement I guess. When you install and start the mobile app for the first time, you are asked to provide the Immich server address.
As of now, you need to give either the IP or the DNS name, followed by  the `/api` suffix/path.
IMHO such details shall not leak and burden the user.
The server address and nothing else should be enough.

**Comment by bo0tzz** [comment]
The server has a [`/.well-known/immich`](https://github.com/immich-app/immich/blob/8b6a79ad9e6dedfbe7b7eb5d8ef6239ea45d36fd/server/src/immich/controllers/app.controller.ts#L11) endpoint that should get [used by the mobile app](https://github.com/immich-app/immich/blob/8b6a79ad9e6dedfbe7b7eb5d8ef6239ea45d36fd/mobile/lib/shared/services/api.service.dart#L110) when you provide it with the base address of the Immich server. Does that not work?

**Comment by juxeii** [comment]
I haven't tested it, but I see that `/api` is suggested on startup. Why give such an endpoint when it is not needed?
I is just a distraction IMHO.
